docs/website/src/site/categories/components/component-table.njk

```diff
@@ -0,0 +1,110 @@
+{% for component in collections.components -%}
+  {% for module in custom-elements.custom-elements.modules -%}
+    {% if page.url == component.url %}
+      {% for declaration in module.declarations -%}
+        {% if declaration.tagName == component.data.componentTagName %}
+          {% if declaration.members.length %}
+<section>
+  <h2 id="properties" tabindex="-1"><a class="header-anchor" href="#properties">Properties</a></h2>
+  <div class="ds-table-wrapper">
+    <table class="ds-table"><thead>
+      <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Default</th>
+        <th class="ds-table__description">Description</th>
+      </tr></thead>
+      <tbody>
+      {% for member in declaration.members -%}{% if member.privacy %}{% else %}{% if member.kind == "field" %}<tr>
```

Comment:
this looks like an empty if statement, can't you switch the assertion around?

Reply:
The negation was not working and I couldn't find a better solution in this case.

Reply:
Changed :)